frontend/javascripts/dashboard/advanced_dataset/dataset_table.tsx

```diff
@@ -657,7 +657,7 @@ export function DatasetTags({
     updatedDataset: APIMaybeUnimportedDataset,
     shouldAddTag: boolean,
     tag: string,
-    event: React.MouseEvent,
+    event: React.SyntheticEvent,
```

Comment:
This callback is used in multiple props. Some are `MouseEvent`s,  some are `ChangeEvent`s, ...